src/components/20-molecules/accordion/CHANGELOG.md

```diff
@@ -0,0 +1,3 @@
+## 1.1.0
+
+Add on state change listener for react and native #2261
```

Comment:
on state change listener -> onStateChange callback and axa-state-change custom event